rollup.config.js

```diff
@@ -32,8 +32,8 @@ export const appConfiguration = {
   },
   plugins: [
     replace({
-      "process.env.G_CLIENT_ID": JSON.stringify(process.env.CLIENT_ID),
-      "process.env.G_API_KEY": JSON.stringify(process.env.API_KEY),
+      "process.env.CLIENT_ID": JSON.stringify(process.env.CLIENT_ID),
+      "process.env.API_KEY": JSON.stringify(process.env.API_KEY),
```

Comment:
```suggestion
      CLIENT_ID: JSON.stringify(process.env.CLIENT_ID),
      API_KEY: JSON.stringify(process.env.API_KEY),
```
This way we don't have to reassign it later on `GoogleSheetsInput.svelte`.

Reply:
Works for me! I'll let you merge #228 and then I'll make my PR conform accordingly